<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QCheckBox" name="cbKeikoku">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Кэйкоку</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHansoku1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Хансоку-1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHansoku2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Хансоку-2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHansoku3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Хансоку-3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
